templates/rapidsai-core/partials/install_rapids_env_pkg.j2

```diff
@@ -0,0 +1,18 @@
+{% if "amd64" in arch %}
```

Comment:
Is this logic only being used in `templates/rapidsai-core/Devel.dockerfile.j2`? If so, you can probably just in-line it. I try to save partials for code that is going to be used across multiple templates.